### PR TITLE
 #159239015 Improve landing page functionality

### PR DESCRIPTION
**Pull request by ddouglasz**
#### What does this PR do?

- Improve landing page functionality

#### Description of Task to be completed?

- Replace Jumbotron with carousel on the landing page
- Make landing page images links to shops and categories
- Style landing page images to have a zoom effect on hover

#### How should this be manually tested?

- Setup application with instructions in the readme
 - Click on any of the landing page images.

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [#159239015](https://www.pivotaltracker.com/story/show/159239015)

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Comment by Daramola98** [comment]
Good work @ddouglasz, After pulling the branch I noticed the product detail page was not functioning as it is expected you may want to look into that.
screen shot 2018-07-27 at 2 57 47 am

File: imports/plugins/custom/landingPage/client/styles/style.css [hunk 1]
```diff
@@ -12,11 +12,12 @@
 
 .header .container {
   margin-top: 22px;
-  min-height: 350px; }
+  min-height: 350px !important;
```
Comment (Daramola98) [inline imports/plugins/custom/landingPage/client/styles/style.css]: I am guessing this is where the problem is coming from styling a bootstrap class container with important is going to affect all places in the application that makes use of this class

Comment (ddouglasz) [reply imports/plugins/custom/landingPage/client/styles/style.css]: hi @Daramola98 , thank you for the observation, on it at the moment.

**Comment by clintfidel** [comment]
Hi @ddouglasz , awesome work, however, I noticed that where the carousel indicators are have like a gray gradient at both sides, I think it should maintain the color of the image, you may want to look into that, please also address the above comment.

File: imports/plugins/custom/landingPage/client/styles/style.css (hunk 1 above)
Comment (ddouglasz) [reply imports/plugins/custom/landingPage/client/styles/style.css]: hi @Daramola98 , I found the styling causing the issue, I have fixed it. 
image

**Comment by saheedt** [comment]
@clintfidel I do not think it's our call to decide what the carousel should look like. Let the PO see it an decide.

**Pull request merged**